### DISCUSSION TITLE
Invalis UTP characters as glyphs...

### DISCUSSION THREAD

**Discussion by vonschutter** [nvim-tree/nvim-tree.lua#2283] [Q&A]
### Description

It looks like some of the glyphs used in the tree view and the rest of the UI are invalid UTF charcters. One shoes up as a block wiht [e6ad] in it. this appears to be some chinese character and not a folder icone--- or am I missing something completely. 

### Neovim version

```text
NVIM 09.1
```

### Operating system and version

ubuntu 23.04

### nvim-tree version

latest stable

### Minimal config

```lua
examples:  buttons = {
      { "  Find File", "Spc f f", "Telescope find_files" },
      { "󰈚  Recent Files", "Spc f o", "Telescope oldfiles" },
      { "󰈭  Find Word", "Spc f w", "Telescope live_grep" },
      { "  Bookmarks", "Spc m a", "Telescope marks" },
      { "  Themes", "Spc t h", "Telescope themes" },
      { "  Mappings", "Spc c h", "NvCheatsheet" },
    },
  }, th [?] are invalid.
```

### Steps to reproduce

snap install nvim; git clone nvchad to the nvim folder in .config ; let the installs complete... the C-n and look at the file list... not pretty... 

### Expected behavior

I should see the icons on the screenshots. 

### Actual behavior

messed up placeholders for where the glyph should be.

**Comment by alex-courtis** [comment]
> Minimal config

That is not a minimal config. Please see [Clean Room Replication](https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#clean-room-replication)

Is [nvim-web-devicons](https://github.com/nvim-tree/nvim-web-devicons) installed and running? See [Installation](https://github.com/nvim-tree/nvim-tree.lua/wiki/Installation)

See [Troubleshooting](https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#icons-not-correctly-shown) if you are having issues with fonts.
